samza-core/src/test/java/org/apache/samza/table/remote/descriptors/TestRemoteTableDescriptor.java

```diff
@@ -148,42 +146,39 @@ public void testSpecifyBothRateAndRateLimiter() {
   public void testTablePartToConfigDefault() {
     TableReadFunction readFn = createMockTableReadFunction();
     when(readFn.toConfig(any(), any())).thenReturn(createConfigPair(1));
-    Map<String, String> tableConfig = new RemoteTableDescriptor("1")
-        .withReadFunction(readFn)
-        .withReadRateLimit(100)
-        .toConfig(new MapConfig());
+    Map<String, String> tableConfig =
+        new RemoteTableDescriptor("1").withReadFunction(readFn).withReadRateLimit(100).toConfig(new MapConfig());
     verify(readFn, times(1)).toConfig(any(), any());
-    Assert.assertEquals("v1", tableConfig.get("k1"));
+    Assert.assertEquals("v1", tableConfig.get("tables.1.io.read.func.k1"));
   }
 
   @Test
   public void testTablePartToConfig() {
 
-    int keys = 0;
+    int key = 0;
```

Comment:
Why this change?

Reply:
The configs used to be {k0 -> v0, k1 -> v1, ...}. But after the change, it becomes {tables.1.io.read.func.k0 -> v0, tables.1.io.read.func.k1 -> v1, ...}. And the other of the pairs is unpredictable. So I made them all to be <prefix>.k0 -> v0, without increasing the key each time.